src/map/lua/luautils.cpp

```diff
@@ -2237,6 +2238,16 @@ namespace luautils
             PChar->updatemask |= UPDATE_HP;
         }
 
+        if (PChar->TradeContainer->getItemsCount() > 0)
+        {
+            ShowDebug("Event [%d] finished without trade cleaned up. "
+                      "Call player:tradeComplete(false) or player:confirmTrade onEventFinish.", eventID);
```

Comment:
This line is being complained about by CI.  Move the `eventID);` to the next line (see the CI Sanity Check for detail)

Reply:
Done.